src/components/button/button-classic.style.js

```diff
@@ -0,0 +1,342 @@
+import { css } from 'styled-components';
+
+const blue = {
+  default: '#255bc7',
+  hover: '#1e499f',
+  active: '#1963f6'
+};
+
+const grey = {
+  default: '#335c6d',
+  hover: '#003349',
+  active: '#335c6d'
+};
+
+const magenta = {
+  default: '#ed1c5f',
+  hover: '#be164c',
+  active: '#f32e6d'
+};
+
+const magentaDull = {
+  default: '#ca2a60',
+  hover: '#be164c',
+  active: '#d55580'
+};
+
+const red = {
+  default: '#c7384f',
+  hover: '#9f2d3f',
+  active: '#e42c2d'
+};
+
+const white = {
+  default: '#ffffff',
+  hover: '#cccccc',
+  text: '#1b1d21',
+  active: '#e6ebed'
+};
+
+const buttonStyles = {
+  primary: {
+    blue: `
+      background: ${blue.default};
+      border: 1px solid transparent;
+      color: ${white.default};
+      &:active {
+        background-color: ${blue.active};
+        border-color: ${blue.active};
+      }
+      &:hover {
+        background: ${blue.hover};
+        border-color: ${blue.hover};
+      }
+      &:focus {
+        box-shadow: 0 0 6px rgba(25,99,246,.6);
```

Comment:
Should these `box-shadow` colours be in your colour configs?

Reply:
it's classic styling